packages/bot-web-ui/src/assets/sass/toolbox.scss

```diff
@@ -165,4 +110,39 @@ $category-colours: (
         font-weight: bold;
         line-height: 1.4;
     }
+    &__sub-category {
+        &-row {
+            padding: 1rem;
+            cursor: pointer;
+
+            &:hover {
+                background-color: var(--general-hover);
+            }
+        }
+    }
+    .dc-input {
```

Comment:
Is this for search? Isn't it better to put it in the `&__search` class?

Reply:
Yeap, this is done. 👍